src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -22,19 +30,55 @@
    * the database and checking to make sure its valid and not null
    * and sending this information to the webpage
    */
+   
+  /** Store class that gives access to Users. */
```

Comment:
For single line comments like these you can use `//` style. The only time we require `/** */` style comments is for docstrings which are exclusive to classes and functions.